### PR TITLE
Add footer toggle to Onboarding's tag follow slide

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] Feature

## Description
- Essentially applied the work I've done on https://github.com/thepracticaldev/dev.to/pull/7266 to FollowTag slide
- One detail I missed in https://github.com/thepracticaldev/dev.to/pull/7266 is when there's no follow, font-weight of following count message should be 500 instead of 700. I've updated that here
## Related Tickets & Documents
Resolves https://github.com/thepracticaldev/dev.to/issues/6548
## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
Kapture 2020-04-14 at 18 32 37

## Added tests?
- [x] updated existing snapshots

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

File: app/javascript/onboarding/components/FollowTags.jsx [hunk 1]
```diff
@@ -74,30 +74,45 @@ class FollowTags extends Component {
...
       const { next } = this.props;
       next();
     });
   }
 
+  renderFollowCount() {
+    const { selectedTags } = this.state;
+    let followingStatus;
+    if (selectedTags.length < 2) {
+      followingStatus = `${selectedTags.length} topic selected`;
```
Comment (vaidehijoshi) [inline app/javascript/onboarding/components/FollowTags.jsx:87]: What if `selectedTags.length` is `0`? Based on [the snapshot](https://github.com/thepracticaldev/dev.to/pull/7293/files#diff-9560b1a73a8f29f53129b0b6cadf5fd9R270), it looks like it won't pluralize correctly.

File: app/javascript/onboarding/components/FollowTags.jsx [hunk 2]
```diff
@@ -74,30 +74,45 @@ class FollowTags extends Component {
...
+    } else {
+      followingStatus = `${selectedTags.length} topics selected`;
+    }
+
+    const klassName =
```
Comment (vaidehijoshi) [inline app/javascript/onboarding/components/FollowTags.jsx:92]:
```suggestion
    const classStyle =
```
Just a suggestion, it felt really weird to see `klass` in JavaScript 😱

**Comment by maestromac** [comment]
@rhymes I was just following the design spec but you raised a very good question. I think it's best to stick with existing terminology instead of creating a new one so I'm going to switch `topics` to `tags`

File: app/javascript/onboarding/components/FollowTags.jsx (hunk 1 above)
Comment (maestromac) [reply app/javascript/onboarding/components/FollowTags.jsx]: Nice catch! Updated

File: app/javascript/onboarding/components/FollowTags.jsx (hunk 2 above)
Comment (maestromac) [reply app/javascript/onboarding/components/FollowTags.jsx]: Updated 😆

**Comment by lisasy** [comment]
@rhymes @maestromac 

Good catch on "topic" vs "tags" — this was something that was just in the design mocks, but I think it's a good idea to go with our existing taxonomy in our product - "topics".

I was using "topics" here because it could be possible that in the future, as we move more into Forem communities, we might think might consider adopting another terminology that we think people can better understand. There are some open questions here that we will revisit in the upcoming months, but for now, keeping it "tags" is great. 👍🏽

**Pull request merged**